containers/Ideas/ExplorePage.js

```diff
@@ -29,7 +21,10 @@ export const ExplorePage = (props) => {
   );
 
   useEffect(() => {
-    fetchProjects();
+    const projectsListener = props.navigation.addListener('focus', () => {
```

Comment:
nice! i guess this is v5.x?

Reply:
ah yes the commit message says it `...make listeners work with new navigation` 💁🏾‍♂️